schema.d.ts

```diff
@@ -40,6 +40,11 @@ export interface Package {
    * An array of the modules this package contains.
    */
   modules: Array<Module>;
+
+  /**
+   * Is this package deprecated? Provide a string to give a reason or migration instructions.
```

Comment:
I think descriptions should be noun-phrases and written for consumers of the objects. Something like "Whether the package is deprecated. If the value is a string, it's the reason for deprecation."